From 3afe2e1c731f5cc7964cd487e660c152299cd2f4 Mon Sep 17 00:00:00 2001
From: "Philip, Avinash" <avinashphilip@ti.com>
Date: Tue, 13 Mar 2012 12:59:55 +0530
Subject: [PATCH 0894/1092] ARM: OMAP2+: am33xx: Enable UBIFS support in
 defconfig

UBIFS is the default file system for NAND MTD devices in am33xx
platforms. Hence enable the defconfig for UBIFS support.

Signed-off-by: Philip, Avinash <avinashphilip@ti.com>
---
 arch/arm/configs/am335x_evm_defconfig | 7 ++++++-
 1 file changed, 6 insertions(+), 1 deletion(-)

diff --git a/arch/arm/configs/am335x_evm_defconfig b/arch/arm/configs/am335x_evm_defconfig
index 5d85b51..e29bf78 100644
--- a/arch/arm/configs/am335x_evm_defconfig
+++ b/arch/arm/configs/am335x_evm_defconfig
@@ -2296,7 +2296,12 @@ CONFIG_MISC_FILESYSTEMS=y
 # CONFIG_BFS_FS is not set
 # CONFIG_EFS_FS is not set
 # CONFIG_JFFS2_FS is not set
-# CONFIG_UBIFS_FS is not set
+CONFIG_UBIFS_FS=y
+# CONFIG_UBIFS_FS_XATTR is not set
+# CONFIG_UBIFS_FS_ADVANCED_COMPR is not set
+CONFIG_UBIFS_FS_LZO=y
+CONFIG_UBIFS_FS_ZLIB=y
+# CONFIG_UBIFS_FS_DEBUG is not set
 # CONFIG_LOGFS is not set
 CONFIG_CRAMFS=y
 # CONFIG_SQUASHFS is not set
-- 
1.7.11.2

